contrib/mpd/template.py

```diff
@@ -0,0 +1,73 @@
+pkgname = "mpd"
+pkgver = "0.23.15"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Ddocumentation=enabled",
+    "-Dhtml_manual=true",
+    "-Dmanpages=true",
+    "-Dipv6=enabled"
+]
+hostmakedepends = ["pkgconf", "meson", "python-sphinx"]
+# MikMod causes a crash
+makedepends = [
+    "alsa-lib-devel",
+    "audiofile-devel",
+    "avahi-devel",
+    "boost-devel",
+    "bzip2-devel",
+    "chromaprint-devel",
+    "fdk-aac-devel",
+    "ffmpeg-devel",
+    "flac-devel",
+    "fluidsynth-devel",
+    "fmt-devel",
+    "lame-devel",
+    "libcdio-paranoia-devel",
+    "libcurl-devel",
+    "libexpat-devel",
+    "libgme-devel",
```

Comment:
pass explicit meson -Dsomething=enabled for these if possible

Reply:
(like most of these libs, not just these lines, i commented in a random spot here)